### PR TITLE
fix links

### PR DESCRIPTION
**Pull request by septs**
https://github.com/tc39/dataset/actions/runs/4580104991

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,9 +183,6 @@ Note that as part of the onboarding process your repository name may be normaliz
 [module-declarations-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2022-11/dec-01.md#module-declarations
 [symbol-predicates]: https://github.com/tc39/proposal-symbol-predicates
 [symbol-predicates-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2023-01/feb-01.md#symbol-predicates
-[iterator-range]: https://github.com/tc39/proposal-Number.range
-[iterator-range-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-07/july-22.md#numberrange-for-stage-2
```
Comment (septs) [inline README.md:187]: unused link definiton

Comment (ljharb) [reply README.md:187]: how is this unused? i see it used on line 70.

File: README.md [hunk 2]
```diff
@@ -183,9 +183,6 @@ Note that as part of the onboarding process your repository name may be normaliz
...
+[iterator-range]: https://github.com/tc39/proposal-iterator.range
 [float16s]: https://github.com/tc39/proposal-float16array
-[float16s-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2017-05/may-23.md#16ig-float16-on-typedarrays-dataview-mathhfround-for-stage-1
 [async-context]: https://github.com/tc39/proposal-async-context
-[async-context-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2023-01/feb-01.md#async-context-for-stage-1
```
Comment (ljharb) [inline README.md:191]: please leave the notes placeholders here, even though they're unused.

Comment (ljharb) [inline README.md:191]: please leave the notes placeholders here, even though they're unused - that's intentional.

**Review by ljharb — CHANGES_REQUESTED** [review]

**Review by ljharb — APPROVED** [review]
took care of it. thanks!

File: README.md (hunk 2 above)
Comment (septs) [reply README.md]: but you also reverted, `proposal-Number.range` -> `proposal-iterator.range` fixed

Comment (ljharb) [reply README.md]: whoops, missed that. thanks, will fix.

Comment (ljharb) [reply README.md]: fixed in 4232f6089a0c162817724cd418dc0c233dfb6834

**Pull request merged**